### PR TITLE
Certifier attendant

### PR DESCRIPTION
**Pull request by ptran265**
- new birth certifier and birth attendant practitioner profiles, updated change log and content transitions
- a little clean up and updated scripts to point to VRCL instead of sandbox

### CODE REVIEW

File: input/fsh/instances/encounter-birth-babyg-quinn.fsh [hunk 1]
```diff
@@ -14,10 +14,10 @@ Usage: #example
 * participant[0]
   * type = $loinc#87287-9 "Birth certifier [Extended Identifier]"
   * period.start = "2019-02-12T13:30:00-07:00"
-  * individual = Reference(practitioner-vital-records-avery-jones) "Practitioner - Vital Records (Avery Jones, MD)"
+  * individual = Reference(practitioner-vital-records-avery-jones) "Practitioner - Certifier (Avery Jones, MD)"
```
Comment (saulakravitz) [inline input/fsh/instances/encounter-birth-babyg-quinn.fsh]: The referenced practitioner hasn't changed, and indeed the practitioner should conform to:
1) Practitioner Vital Records ("base profile")
2) Certifier
3) Attendant

Bot the references point to the same instance, and I think the instance is still a Practitioner Vital Records, so I would undo the changes in the display name.

File: input/fsh/instances/encounter-maternity-carmen-teresa-lee.fsh [hunk 2]
```diff
@@ -13,7 +13,7 @@ Usage: #example
 * subject = Reference(patient-mother-carmen-teresa-lee) "Patient - Mother (Carmen Teresa Lee)"
 * participant
   * type = $loinc#87286-1 "Birth attendant [Extended Identifier]"
-  * individual = Reference(practitioner-vital-records-janet-seito) "Practitioner - Vital Records (Janet Seito, MD)"
```
Comment (saulakravitz) [inline input/fsh/instances/encounter-maternity-carmen-teresa-lee.fsh:16]: Janet Seito, like Avery Jones, is a 'Practitioner - Vital Records'.

File: input/fsh/profiles/PractitionerAttendant.fsh [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+* name.text ^short = "Attendant's name"
+* identifier ^short = "Note: required by USCore"
+* identifier[NPI] ^short = "Attendant's NPI Number"
+* qualification 0..1
+* qualification 
+  * ^slicing.discriminator.type = #value
+  * ^slicing.discriminator.path = "code"
+  * ^slicing.rules = #open
+  * ^slicing.description = "Slicing based on the profile conformance of the sliced element"
+* qualification contains birthAttendantQualification 0..1
```
Comment (saulakravitz) [inline input/fsh/profiles/PractitionerAttendant.fsh:15]: Is the qualification needed in both the Attendant and Certifier profiles?

File: input/images/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv [hunk 4]
```diff
@@ -330,7 +330,7 @@ U = Unknown",,VRDR,Decedent,extension[NVSS-SexAtDeath] ,codeable,[Administrative
 478,Natality,14,55,4,Date of Birth (Mother)--Year,MDOB_YR,4 digit year (< year of birth of child);  9999 = unknown,,VRCPL,PatientMotherVitalRecords,birthDate,date,See [PartialDatesAndTimes],B,,,,,Patient-mother-vr,
 831,Fetal Death,14,55,4,Date of Birth (Mother)--Year,MDOB_YR,"4 digit year; < year of delivery, 9999",,VRCPL,PatientMotherVitalRecords,birthDate,date,See [PartialDatesAndTimes],B,,,,,Patient-mother-vr,
 1355,Birth Infant Death,14,55,4,Date of Birth (Mother)--Year,MDOB_YR,4 digit year (< year of birth of child);  9999 = unknown,,,,,,,,,,,,,
-,Fetal Death,NA,NA,NA,Method of Disposition,*NO IJE MAPPING*,,,BFDR,ObservationFetalRemainsDispositionMethod,value,codeable,[FetalRemainsDispositionMethodVS],,,,,,,
+,Fetal Death,NA,NA,NA,Method of Disposition,*NO IJE MAPPING*,,,BFDR,ObservationMethodOfDisposition,value,codeable,[MethodOfDispositionVS],,,,,,,
```
Comment (saulakravitz) [inline input/images/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv]: I thought we changed MethodOfDisposition --> FetalRemainsDispositionMethod....  
So, I think this is not correct.

File: input/images/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv [hunk 5]
```diff
@@ -828,7 +828,7 @@ D = Donation
 E = Entombment
 R = Removal from state
 O = Other
-U = Unknown",,VRDR,DecedentDispositionMethod,value,codeable,[FetalRemainsDispositionMethodVS],,y,,,,DecedentDispositionMethod,
```
Comment (saulakravitz) [inline input/images/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv:831]: I don't think this is right

File: input/mapping/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv [hunk 6]
```diff
@@ -330,7 +330,7 @@ U = Unknown",,VRDR,Decedent,extension[NVSS-SexAtDeath] ,codeable,[Administrative
 478,Natality,14,55,4,Date of Birth (Mother)--Year,MDOB_YR,4 digit year (< year of birth of child);  9999 = unknown,,VRCPL,PatientMotherVitalRecords,birthDate,date,See [PartialDatesAndTimes],B,,,,,Patient-mother-vr,
 831,Fetal Death,14,55,4,Date of Birth (Mother)--Year,MDOB_YR,"4 digit year; < year of delivery, 9999",,VRCPL,PatientMotherVitalRecords,birthDate,date,See [PartialDatesAndTimes],B,,,,,Patient-mother-vr,
 1355,Birth Infant Death,14,55,4,Date of Birth (Mother)--Year,MDOB_YR,4 digit year (< year of birth of child);  9999 = unknown,,,,,,,,,,,,,
-,Fetal Death,NA,NA,NA,Method of Disposition,*NO IJE MAPPING*,,,BFDR,ObservationFetalRemainsDispositionMethod,value,codeable,[FetalRemainsDispositionMethodVS],,,,,,,
```
Comment (saulakravitz) [inline input/mapping/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv]: I don't think this is right.

File: input/mapping/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv [hunk 7]
```diff
@@ -828,7 +828,7 @@ D = Donation
 E = Entombment
 R = Removal from state
 O = Other
-U = Unknown",,VRDR,DecedentDispositionMethod,value,codeable,[FetalRemainsDispositionMethodVS],,y,,,,DecedentDispositionMethod,
+U = Unknown",,VRDR,DecedentDispositionMethod,value,codeable,[MethodOfDispositionVS],,y,,,,DecedentDispositionMethod,
```
Comment (saulakravitz) [inline input/mapping/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv:831]: I don't think this is right.

File: input/pagecontent/ije_mapping_fetalDeath.md [hunk 8]
```diff
@@ -180,7 +180,6 @@ All of the none-of-the-above values are represented as observations with a clear
...
 <tr><td style='text-align: center;'>109</td><td>Date Last Normal Menses Began--Day</td><td style='text-align: center;'>DLMP_DY</td><td><a href='StructureDefinition-Observation-last-menstrual-period.html'>ObservationLastMenstrualPeriod</a></td><td>value</td><td style='text-align: center;'>dateTime</td><td>See <a href='https://hl7.org/fhir/us/vr-common-library/2024Jan/usage.html#partial-dates-and-times'>PartialDatesAndTimes</a></td></tr>
-<tr><td style='text-align: center;'>NA</td><td>Method of Disposition</td><td style='text-align: center;'>*NO IJE MAPPING*</td><td><a href='StructureDefinition-Observation-fetal-remains-disposition-method.html'>ObservationFetalRemainsDispositionMethod</a></td><td>value</td><td style='text-align: center;'>codeable</td><td><a href='ValueSet-ValueSet-fetal-remains-disposition-method.html'>FetalRemainsDispositionMethodVS</a></td></tr>
```
Comment (saulakravitz) [inline input/pagecontent/ije_mapping_fetalDeath.md]: I don't think this is right

**Review by saulakravitz — CHANGES_REQUESTED** [review]
See my comments regarding the instances of the practitioners, and the fetal remains dispoaition method.

File: input/images/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv (hunk 4 above)
Comment (ptran265) [reply input/images/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv]: I think I was using an outdated version of the spreadsheet... thanks for catching that

File: input/fsh/profiles/PractitionerAttendant.fsh (hunk 3 above)
Comment (ptran265) [reply input/fsh/profiles/PractitionerAttendant.fsh:15]: Yes, I believe so. That was an oversight when we pulled it from PractitionerVR, but I will add it to Certifier

File: input/images/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv (hunk 5 above)
Comment (ptran265) [reply input/images/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv:831]: This is VRDR DecedentDispositionMethod which uses MethodOfDispositionVS

File: input/mapping/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv (hunk 7 above)
Comment (ptran265) [reply input/mapping/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv:831]: Same as above- this is for VRDR which uses different VS

**Review by saulakravitz — APPROVED** [review]
